assets/js/googlesitekit/datastore/site/index.test.js

```diff
@@ -25,9 +25,11 @@ import apiFetch from '@wordpress/api-fetch';
  * Internal dependencies
  */
 import { createTestRegistry, unsubscribeFromAll } from 'tests/js/utils';
-import { INITIAL_STATE } from './index';
+import { default as datastore } from './index';
```

Comment:
I think this can be simplified to `import datastore from './index';`? Maybe let's name it `store` for consistency with the other ones.

Reply:
Ah yes. I used `dataStore` because `store` was used in the inner scope. Bit since we only need it to get `INITIAL_STATE`, we should be able to override it.